openfasoc/generators/ldo-gen/flow/scripts/save_images.tcl

```diff
@@ -20,7 +20,8 @@ gui::set_display_controls "Instances/StdCells/*" visible true
 gui::set_display_controls "Instances/Macro" visible true
 gui::set_display_controls "Instances/Pads/*" visible true
 gui::set_display_controls "Instances/Physical/*" visible true
-gui::set_display_controls "Pin Markers" visible true
+gui::set_display_controls "Shape Types/Pins" visible true
+gui::set_display_controls "Shape Types/*/*" visible true
```

Comment:
Can line 23 also be done in the same way as 24, in case something changes again upstream

Reply:
Using `Shape Types/*`?